code/game/machinery/camera/tracking.dm

```diff
@@ -25,9 +25,10 @@
 	var/list/T = list()
 
 	for(var/obj/machinery/camera/C in L)
-		var/list/tempnetwork = C.network & src.network
-		if(tempnetwork.len)
-			T[text("[][]", C.c_tag, (C.can_use() ? null : " (Deactivated)"))] = C
+		if(C.network)
+			var/list/tempnetwork = C.network & network
+			if(tempnetwork && tempnetwork.len)
+				T[text("[][]", C.c_tag, (C.can_use() ? null : " (Deactivated)"))] = C

```

Comment:
Huh, I wonder why some camera machines don't get their network set if the AI latejoins... Well, more sanity is good, at least. Should the case of a camera lacking a network be logged, so that we can isolate the issue, like with the null prefs muting bug?